lib/matplotlib/colorbar.py

```diff
@@ -670,35 +643,35 @@ def _do_extends(self, extendlen):
         if self._extend_lower():
             if not self.extendrect:
                 # triangle
-                xy = np.array([[0.5, 0], [1, elower], [0, elower]])
+                xy = np.array([[0.5, bot], [1, 0], [0, 0]])
             else:
                 # rectangle
-                xy = np.array([[0, 0], [1., 0], [1, elower], [0, elower]])
+                xy = np.array([[0, bot], [1., bot], [1, 0], [0, 0]])
```

Comment:
Is there a reason these are not in the same order as above (starting at `[0, 0]`)?

Reply:
just more parallel to the triangle case...

Reply:
I did mean the order of _both_ triangle and rectangle cases.

Reply:
sure